chapter05/src/main/java/com/seaofnodes/simple/node/ScopeNode.java

```diff
@@ -19,7 +19,7 @@ public class ScopeNode extends Node {
     public final Stack<HashMap<String, Integer>> _scopes = new Stack<>();
 
     @Override
-    public String label() { return "Scope" + _nid; }
+    public String label() { return "Scope"; }
```

Comment:
isn't this going to break the Graph Visualizer? We needed distinct labels to be able to draw separate clusters